packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes } from 'react';
+import {
+	extractSpecialFields,
+	getId,
+	getLabel,
+	renderCell,
+} from '../utils/gridrow';
+
+import theme from './RowLarge.scss';
+
+/**
+ * Row renderer that displays a Large item
+ */
+function RowLarge({ className, index, key, parent, style }) {
+	const { titleField, selectionField, otherFields } = extractSpecialFields(parent);
+
+	const parentId = getId(parent);
+	const id = parentId && `${parentId}-${index}`;
+	const titleCell = titleField && renderCell(index, parent, titleField);
+	const selectionCell = selectionField && renderCell(index, parent, selectionField);
+	const otherCellsListItems = otherFields.map((field, fieldIndex) => {
+		const cellContent = renderCell(index, parent, field);
+		const tooltip = typeof cellContent === 'string' ? cellContent : null;
+		const label = getLabel(field);
+		return (
+			<li key={fieldIndex}>
+				{label && (<span className={theme['field-label']}>{label}: </span>)}
+				<span className={theme['field-value']} title={tooltip}>{cellContent}</span>
+			</li>
+		);
+	});
```

Comment:
Agree on the fact it cost.
disagree on the memoized.
We may in that case prefer to use a class with ShouldComponentUpdate. Will do that later.